packages/generate-component-docs/src/generate.ts

```diff
@@ -242,7 +244,7 @@ function extractTypeInfo(file: string, options: CompilerOptions) {
         checker.typeToString(unionItem),
       );
 
-      if (isEqual(types.slice(0, 7), unionAliases.ReactNode)) {
+      if (isEqual(types.slice(0, 9), unionAliases.ReactNode)) {
```

Comment:
Might be nice to use the length of the array for this slice to make their relationship explicit.
```suggestion
      if (
        isEqual(
          types.slice(0, unionAliases.ReactNode.length),
          unionAliases.ReactNode,
        )
      ) {
```